packages/story-editor/src/components/form/color/color.js

```diff
@@ -52,7 +52,6 @@ const containerCss = css`
 const Container = styled.section`
   ${containerCss}
   gap: ${({ isInDesignMenu }) => (isInDesignMenu ? 6 : 8)}px;
-  width: ${({ width }) => (width ? `${width}px` : `100%`)};
```

Comment:
If we remove the `width` override here, `width` should no longer be needed as a prop. BUT I added this because we were getting weird sizing on the input on other use cases in the floating menu without it.
<img width="708" alt="Screen Shot 2022-03-22 at 4 03 52 PM" src="https://user-images.githubusercontent.com/1820266/159584056-238f41c7-663b-4c69-a56b-64a40f51f978.png">

Reply:
ah drag!